src/idiom128.fuzion

```diff
@@ -0,0 +1,8 @@
+BreadthFirstTraverse(f node -> unit, root node) =>
+  queue := [root].asList
+  BreadthFirstTraverse f queue
+  BreadthFirstTraverse(f node -> unit, queue List<node>) =>
+    if queue.count > 0 then
```

Comment:
Calling count here results in quadratic performance since count really counts the elements. So if count is, e.g., 1000, we count these 1000, then we count 999, 998, 997, 996, ..., in total 500500 elements!
Better use
```
match queue
  nil -> 
  c Cons -> 
    f c.head
    BreadthFirstTraverse f (c.tail ++ t.children)
```